erts/etc/unix/erl.src.src

```diff
@@ -15,10 +15,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# 
+#
 # %CopyrightEnd%
 #
-ROOTDIR="%FINAL_ROOTDIR%"
+if [ -z "$ROOTDIR" ]
+then
+    ROOTDIR="%FINAL_ROOTDIR%"
+fi
+
+# If ROOTDIR is not set or if it doesn't point to an existing directory, try
+# to find the path of the script from within itself to set ROOTDIR dynamically
+if [ -z "$ROOTDIR" -o ! -d "$ROOTDIR" ]
+then
+    prog="$0"
+    # Follow all symlinks, whether relative or absolute, leading to the script
+    while [ -h "${prog}" ]; do
```

Comment:
The code to find the path to the script seems to be inspired by [this](https://android.googlesource.com/platform/dalvik/+/master/dx/etc/dx) as you mention in [ERL-1323](https://bugs.erlang.org/browse/ERL-1323). I think it would be appropriate if you add a comment with a reference to the dx code.

Reply:
Great suggestion, I'll do this in my next revision